### PR TITLE
feat: bump up Renderer to Renderer2

### PR DESCRIPTION
**Pull request by wKoza**
For complete the PR #1512 , Renderer should be replaced by Renderer2.

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Awesome, thnx @wKoza - I was planning to do this one!

File: src/popover/popover.spec.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ describe('ngb-popover', () => {
       directive.triggerEventHandler('click', {});
       fixture.detectChanges();
       expect(getWindow(fixture.nativeElement)).toBeNull();
-      expect(directive.nativeElement.getAttribute('aria-describedby')).toBeNull();
+      expect(directive.nativeElement.getAttribute('aria-describedby')).toBe('null');
```
Comment (pkozlowski-opensource) [inline src/popover/popover.spec.ts]: Hmm, this doesn't sound right. It seems like here we would have `aria-describedby="null"` while what we want is no attribute at all. Was this change necessary after migration to the new renderer?

Comment (wKoza) [inline src/popover/popover.spec.ts]: Outch, Ok I see. I fix this

**Comment by pkozlowski-opensource** [comment]
LGTM, thnx!
